Makefile.toml

```diff
@@ -1255,8 +1255,8 @@ set -e
 
 export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
 
-if [ -z "${GRANT_TO_USERS}" ] && [ -z "${GRANT_TO_GROUPS}" ]; then
-   echo "GRANT_TO_USERS and/or GRANT_TO_GROUPS is mandatory for grant-ami; please give a comma-separated list of user IDs or group names" >&2
+if [ -z "${GRANT_TO_USERS}" ] && [ -z "${GRANT_TO_GROUPS}" ] && [ -z "${GRANT_TO_ORGS}" ] && [ -z "${GRANT_TO_OUS}" ]; then
```

Comment:
Longer, but would it be better to make it readable? `GRANT_TO_ORG_UNITS`?

Reply:
+1 to this. As someone still new to a lot of AWS terminology, I would not know with `OUS` means.